reference/physics.md

```diff
@@ -95,8 +95,7 @@ necessary to specify all the [Physics](#physics) nodes in a Webots world.
 Whether to use or not a [Physics](#physics) node in a particular case depends on
 what aspect of the real world your want to model in your simulation.
 
-Note that if a [Solid](#solid) node contains a [Physics](#physics) node but none of its ancestor nodes contain a [Physics](#physics) node, then this node will be simulated in *physics* mode and it will move independently from its ancestor nodes.
-This is not true for the [TouchSensor](#touchsensor) node. If all the ancestor nodes contain no [Physics](#physics) node, the [TouchSensor](#touchsensor) is attached to the static environment with an ODE fixed joint and thus it won't move from its initial position.
+Note that if a [Solid](#solid) node contains a [Physics](#physics) node but none of its ancestor nodes contain a [Physics](#physics) node, then this node will be attached to the static environment with an ODE fixed joint and thus it won't move from its initial position.
```

Comment:
No, because the top Ball solid is not a kinematic object, i.e. the top Solid with physics has no parent Solid (with or without physics).

I didn't changed anything to the way solid are attached to the static object w.r.t to the previous Webots implementation. But it is good to describe this behavior in the documentation.